js/browser.ts

```diff
@@ -0,0 +1,5 @@
+export const fetch: typeof window.fetch = window.fetch;
```

Comment:
Can TypeScript not narrow/infer (whatever we call it?) these?

Reply:
Looks like it can, I forget why I originally included that. ecbdcac